src/main/java/at/hannibal2/skyhanni/test/command/CopyChestNameCommand.kt

```diff
@@ -0,0 +1,15 @@
+package at.hannibal2.skyhanni.test.command
+
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.OSUtils
+
+object CopyChestNameCommand {
```

Comment:
Why would someone need the command? just for skytils? then using the hotkey in skyhanni makes more sense imo

Reply:
>Why would someone need the command?
neu inventory buttons